ember-scoped-css/package.json

```diff
@@ -64,6 +68,7 @@
     "test": "vitest"
   },
   "dependencies": {
+    "@ast-grep/napi": "^0.39.9",
```

Comment:
I had this as devDep at first, but this is installing the binaries for each os. That's why then the build was failing (for `ember-scoped-css`).

The alternative is to make it a peer dep. Then optional? But `rolldown` users have to install it.